insight-rest-objects/src/main/java/org/nuxeo/ai/sdk/objects/PropertyType.java

```diff
@@ -25,7 +25,14 @@
 import java.util.Objects;
 
 /**
- * POOJO for defining property -> type relation
+ * POJO for defining property -> type relation
+ *
+ * <code>
+ * {
+ *   "name": "file:content",
+ *   "type": "img"
+ * }
```

Comment:
This feels like a representation needed for a DocumentType in Nuxeo. Do we need this on the Client ? so many other places where this kind of information is .

Reply:
that's not the issue of this type. that's the issue of our design

Reply:
The data documents to communicate between client and insights do not need to represent exactly what is used internally. We can make them pretty/readable here.

Reply:
As in, not a documentation issue, but object definition .